ui/Screen/Project/Source/MergeRequestButton.svelte

```diff
@@ -25,35 +18,51 @@ git push --tags rad`;
     border-radius: 0.5rem;
     background: var(--color-background);
     border: 1px solid var(--color-foreground-level-3);
-    box-shadow: var(--elevation-medium);
+    box-shadow: var(--color-shadows);
     padding: 1rem;
     width: 25rem;
   }
 
   p {
     color: var(--color-foreground-level-6);
     user-select: none;
+    margin-bottom: 1rem;
+  }
+
+  p:last-child {
+    margin-bottom: 0;
   }
 </style>
 
 <Overlay {expanded} on:hide={hide} style="position: relative;">
   <div class="request-dropdown" hidden={!expanded}>
-    <p style="margin-bottom: 0.5rem;">
-      To create a new merge request, run this in your terminal:
+    <p>1. Make your changes.</p>
+    <p>
+      2. Annotate a git tag with the title
+      <span class="typo-mono-bold">{`merge-request/<name>`}</span>. Be sure to
+      replace the
+      <span class="typo-mono-bold">{`<name>`}</span>
+      with your Merge Request title.`}
```

Comment:
I’d phrase this differently because the `name` is not the title of the merge request: it’s the name or the ID of the tag and merge request.
> Create an annotated Git Tag that starts with `merge-request/`
>  ~~~
> git tag --annotate `merge-request/<name>`
> ~~~
> Be sure to replace `<name>` with the name of your merge request.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1721/commits/4992c97957f1f0564684bcc72847d3fd124cd296

Reply:
I think there was a misunderstanding here. I was suggesting to avoid the term “title” in step . because we also use that term in step 3 to mean something different. To prevent confusion I was suggesting using “name” instead of “title” in step 2.

Reply:
I'm thoroughly confused. Could you make a suggestion?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1721/commits/64e3891fad97ed57da220f03eadf70d454aa4b6b